module.txt

```diff
@@ -4,7 +4,10 @@
     "author": "Josharias",
     "displayName": "Eventual Skills",
     "description": "A library for adding time based skill leveling.",
-    "dependencies": [],
+    "dependencies": [
+        { "id": "CoreAssets", "minVersion": "2.0.0" },
+        { "id": "Notifications", "minVersion": "0.1.0", "maxVersion": "0.3.0" }
```

Comment:
(same as https://github.com/Terasology/InGameHelp/pull/11/files#r541957447)

Hm, I don't like this, but it would get quite hairy for the local dev setup otherwise 🧐 

**The problem:** Notifications is published in v0.1.0 (`master` branch state) and snapshots are published for v0.2.0-SNAPSHOT (`develop` branch). Since we are in the v0.x version range, this is treated as major version, and thus excluded from the default dependency range.

I suspect this would also break when blindly building the Omega line-up which would choose the latest SNAPSHOT version...